src/styles/dynamaic-variables.scss

```diff
@@ -76,6 +76,16 @@
   --playkit-tone-7-color: #212121;
   --playkit-tone-8-color: #000000;
 
+  // Tone Ramp - rgb values
+  --playkit-tone-1-color-rgb: 255, 255, 255;
```

Comment:
so what is the difference in the current color definition vs what we had before the change ?

Reply:
before it was exported only as HEX format and now they are exported as RGB format and whenever you want more control over or be able to manipulate the color element (opacity for example ) you use this representation

read the full desc on the pr desc